examples/groups/group.yaml

```diff
@@ -4,9 +4,10 @@ metadata:
   name: example-group
 spec:
   forProvider:
-    parentId: <parent-group-id>
-    path: "example-path"
-    description: "example description"
+    parentIdRef:
+      name: example-group

```

Comment:
suggestion: 
`name: example-parent-group`
to make it clear it's a different group we are referencing.